src/test/java/com/homework/MyArrayListTest.java

```diff
@@ -0,0 +1,51 @@
+package com.homework;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+public class MyArrayListTest {
+    ArrayList arrayList;
+
+    @Before
+    public void setUp() {
+        arrayList = new ArrayList();
+        arrayList.add(1);
+        arrayList.add(2);
+    }
+
+    @Test
+    public void add() {
+        arrayList.add(3);
+        arrayList.add(4);
+
+        assertEquals(3, arrayList.get(2));
+        assertEquals(4, arrayList.get(3));
+    }
+
+    @Test
+    public void remove() {
```

Comment:
better check size changing here or change logic of this test in another way
I mean even you dont remove(1), assertEquals(1, arrayList.get(0)); always will be true in your case